app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+@import uk.gov.hmrc.helptosavefrontend.controllers.routes
+@import uk.gov.hmrc.helptosavefrontend.views.html.helpers.{form,submit}
+@import views.html.helper.CSRF
+
+@(backupEmail: String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
+
+
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(
+    title = messages("hts.cannot-change-email.page-title"),
+    sidebarLinks = None,
+    bodyClasses = None) {
+
+    <h1 class="heading-xlarge">@Messages("hts.cannot-change-email.title")</h1>
+
+    <p class="lede">@Messages("hts.cannot-change-email.p1")</p>
+
+    @form(routes.NewApplicantUpdateEmailAddressController.verifyEmailErrorSubmit){
+        @CSRF.formField(request)
+        <div class="form-group">
+            <fieldset>
+                <legend>
+                    <h1 class="heading-medium">
+                    @Messages("hts.cannot-change-email.header1", backupEmail)
+                    </h1>
+                </legend>
+
+                <div class="multiple-choice">
+                    <input id="radio-inline-1" type="radio" name="radio-inline-group" value=true>
+                    <label for="radio-inline-1">
+                        @Html(Messages("hts.cannot-change-email.opt1", backupEmail))
+                    </label>
+                </div>
+
+                <div class="multiple-choice">
+                    <input id="radio-inline-2" type="radio" name="radio-inline-group" value=false>
+                    <label for="radio-inline-2">
+                        @Messages("hts.cannot-change-email.opt2")
+                    </label>
+                </div>
+            </fieldset>
+
+        </div>
+
+        @submit('_buttonClass -> "", 'id → "email-error-continue"){ Continue }
```

Comment:
The string `Continue` looks like it belongs in the messages file

Reply:
Yeah and line 49 in file select_email.scala.html is

                <input id="add-new-email" type="radio" name="email" value="No" @if(hasInvalidEmail){ checked }>

so checked should also be in messages?

Reply:
`checked` is not a string here - it's an html attribute